src/NServiceBus.AcceptanceTests/Recoverability/When_transactions_off_and_delayed_retries_enabled.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability
+{
+    using System;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_transactions_off_and_delayed_retries_enabled : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void Should_throw_on_startup()
+        {
+            Requires.DelayedDelivery();
+
+            var exception = Assert.ThrowsAsync<Exception>(async () => await Scenario.Define<ScenarioContext>()
+                .WithEndpoint<StartedEndpoint>()
+                .Done(c => c.EndpointsStarted)
+                .Run());
+
+            StringAssert.Contains("Delayed retries are not supported when running with TransportTransactionMode.None.", exception.ToString());
```

Comment:
Why not use `exception.Message` here? Why not assert on a specific exception?

Reply:
> Why not use exception.Message here

This was adapted from an existing test, so I just used what was already there.

> Why not assert on a specific exception?

Not sure what you mean by this. I'm just throwing `Exception`, and the test is using `Assert.ThrowsAsync<Exception>`

Are you saying there should be a more specific exception type used?